packages/rest/src/router/metadata.ts

```diff
@@ -261,6 +268,65 @@ export function operation(verb: string, path: string, spec?: OperationObject) {
 
 const paramDecoratorStyle = Symbol('ParamDecoratorStyle');
 
+/**
+ * Check if the given type is `stream.Readable` or a subclasses of
+ * `stream.Readable`
+ * @param type JavaScript type function
+ */
+function isReadableStream(type: Object): boolean {
+  if (typeof type !== 'function') return false;
+  if (type === Stream.Readable) return true;
+  return isReadableStream(Object.getPrototypeOf(type));
```

Comment:
AFAIK, an object is treated as a readable stream if it implements a set of methods like `read` and `pipe`. In fact, [this blog post](https://r.va.gg/2014/06/why-i-dont-use-nodes-core-stream-module.html) advocates against using the native ReadableStream implementation (because its behavior changes depending on Node.js version) and recommends to use [readable-stream](https://www.npmjs.com/package/readable-stream) package instead.

@raymondfeng Have you considered using something like [is-stream](https://www.npmjs.com/package/is-stream) instead?